hw/xbox/nv2a/nv2a_shaders.c

```diff
@@ -246,26 +246,31 @@ static void append_skinning_code(QString* str, bool mix,
     } else {
         qstring_append_fmt(str, "%s %s = %s(0.0);\n", type, output, type);
         if (mix) {
```

Comment:
nit: This `mix` name isn't good anymore. I could rename it to `sum_unity` or something.